source/class.cpp

```diff
@@ -408,6 +408,16 @@ void ClassBinder::request_bindings_and_skipping(Config const &config)
 /// extract include needed for this generator and add it to includes vector
 void ClassBinder::add_relevant_includes(IncludeSet &includes) const
 {
+	string const qualified_name_without_template = standard_name( C->getQualifiedNameAsString() );
+	std::map<string, std::vector<string>> const &per_class_includes= Config::get().per_class_includes();
+
+	if( per_class_includes.count(qualified_name_without_template) ) {
```

Comment:
Right now there is two lookup for `qualified_name_without_template` which is not good. Could you please refactor this and use `map.find` and then examine and use it result. You can find example of this technique here: https://github.com/RosettaCommons/binder/blob/master/source/type.cpp#L358

Reply:
Happy to use the idiom here, but I'd argue that this is less readable and really no more performant that .count/.at. Repeated lookups in small maps are exceptionally fast, as the data structure is sitting in cache for the second lookup.